app/src/main/java/org/mozilla/fenix/ext/NavController.kt

```diff
@@ -13,27 +13,44 @@ import androidx.navigation.Navigator
 import io.sentry.Sentry
 import org.mozilla.fenix.BuildConfig
 
-fun NavController.nav(@IdRes id: Int?, directions: NavDirections) {
-    if (id == null || this.currentDestination?.id == id) {
+fun NavController.nav(
```

Comment:
nit: Since the nav helper is starting to get a little complicated some KDoc comments would be nice